.pre-commit-config.yaml

```
@@ -340,7 +340,8 @@ repos:
         language: pygrep
         name: Check for language that we do not accept as community
         description: Please use "deny_list" or "allow_list"  instead.
-        entry: "(?i)(black|white)[_-]?list"
+        entry: "(?i)(\bmaster\b|\bhe\b|\bshe\b|\bhis\b|\bher\b|
+          \bslave\b|\bsanity\b|\bdummy\b|\bwhite\b|\bblack\b)"
```

```suggestion
        entry: > 
            (?i)
            \bmaster\b|
            \bhe\b|
            \bshe\b|
            \bhis\b|
            \bher\b|
            \bslave\b|
            \bsanity\b|
            \bdummy\b|
            \bwhite\b|
            \bblack\b
```

WDYT? Also we probably should remove dummy and add it in other PR. Also apart from changing this regexp you need to fix all issues in documentation, code and other places affected.

---

I think this will not work because you will get "spaces" as part of the regexp. The right way is also to add "(?x)" to ignore whitespace https://www.regular-expressions.info/freespacing.html

---

Right, good catch @potiuk!